### DISCUSSION TITLE
Which platform / software are you considering for our next venture? MS Word, Google Docs or Others?

### DISCUSSION THREAD

**Discussion by tim-hub** [tim-hub/obsidian-bible-reference#118] [Polls]
I'm contemplating extending the Bible reference feature to other platforms as a plugin as well. Undoubtedly, this endeavor will demand significant effort. Hence, I'm eager to gather your collective insights on this matter. Specifically, which platform do you believe should take precedence for implementation? Your input is highly valued.

**Comment by jmantn** [comment]
macOS menu bar to run system wide with customizable activation option.
